haskell/private/cc_wrapper.py.tpl

```diff
@@ -0,0 +1,812 @@
+#!/usr/bin/env python3
+"""CC toolchain wrapper
+
+Usage: cc_wrapper [ARG]...
+
+Wraps the C compiler of the Bazel CC toolchain. Transforms arguments to work
+around limitations of Bazel and GHC and passes those via response file to the C
```

Comment:
You never define what a “response file” is. This is a good place to do so.

Reply:
Done.